fenix/app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -46,5 +46,27 @@ sealed class Event {
          * Event recording the first time a URI is loaded in Firefox in a 24 hour period.
          */
         object FirstUriLoadForDay : GrowthData("ja86ek")
+
+        /**
+         * Event recording when User is "activated" in first week of usage.
+         * Activated = if the user is active 3 days in their first week and
+         *                  if they search once in the latter half of that week (days 4-7).
+         */
+        object UserActivated : GrowthData("13mh5x")
+    }
+
+    /**
+     * Events for helping track user data for growth events
+     */
+    sealed class ActivatedUserData(val tokenName: String) : Event() {
```

Comment:
Since the `object`s below are only namespaced by `ActivatedUserData` and do not implement it, we should be able to remove this unused `tokenName` property.

It also looks like they were potentially meant to implement `ActivatedUserData`, in which case we should be able to remove the property as well as the filler values that are used below.

Reply:
I removed `ActivatedUserData`, see below comment on `Event.kt`